run-tests.php

```diff
@@ -3669,16 +3654,6 @@ class RuntestsValgrind
     protected $version_3_8_0 = false;
     protected $tool = null;
 
-    public function getVersion(): string
-    {
-        return $this->version;
-    }
-
-    public function getHeader(): string
```

Comment:
This is used here:
https://github.com/php/php-src/blob/52480b3a7933ac99ef0f2c0233eeced3661ddacf/run-tests.php#L926

Reply:
Of course, I don't know how my search failed on this.

I already added it again, sorry.